view/partial/add-trial.hbs

```diff
@@ -46,6 +46,12 @@
                     </label>
                     <input id="trial-end" class="form-control" name="endAt" type="date" required>
                 </div>
+                <div class="form-group">
+                    <label for="target-count">
+                        Target Count
+                    </label>
+                    <input type="number" min="0" name="targetCount">
```

Comment:
Link the label and the input `id="target-count"`
Style the input with bootstrap `class="form-control"`
and make the field `required`